toolkit/tools/imagegen/configuration/packagerepo.go

```diff
@@ -203,6 +227,28 @@ func createCustomRepoFile(fileName string, packageRepo PackageRepo) (err error)
 		return
 	}
 
+	// Write the GPGKey field
+	gpgKey := fmt.Sprintf("gpgkey=%s\n", packageRepo.GPGKeys)
+	_, err = stringBuilder.WriteString(gpgKey)
+	if err != nil {
+		logger.Log.Errorf("Error writing repo GPGKey: %s. Error: %s", gpgKey, err)
+		return
+	}
+
+	// Write the repo GPGCheck field
+	var gpgVal string
+	if packageRepo.GPGCheck {
+		gpgVal = "1"
+	} else {
+		gpgVal = "0"
+	}
+	repoGpgCheck := fmt.Sprintf("gpgcheck=%s\n", gpgVal)
+	_, err = stringBuilder.WriteString(repoGpgCheck)
```

Comment:
Suggestion to reduce the amount of code:
```suggestion
	if packageRepo.GPGCheck {
			_, err = stringBuilder.WriteString(gpgcheck=1\n)
	} else {
			_, err = stringBuilder.WriteString(gpgcheck=0\n)
	}
```